common/lib/container-definitions/BREAKING.md

```diff
@@ -4,6 +4,16 @@
 ### IProxyLoader interface to be removed
 The `IProxyLoader` interface has been deprecated in 0.48 and will be removed in an upcoming release.
 
+## 0.49.1000 Breaking changes
```

Comment:
Should we also add this to the root BREAKING.md file? I'm not sure if people check here as frequently, so it may be good for additional exposure.

Reply:
I will in the next PR that consumes this from client packages.  It looks like this is basically unused, not sure the plan for it long-term.  @tylerbutler ?